controllers/harborcluster_controller.go

```diff
@@ -37,14 +38,100 @@ type HarborClusterReconciler struct {
 // +kubebuilder:rbac:groups=goharbor.goharbor.io,resources=harborclusters/status,verbs=get;update;patch
 
 func (r *HarborClusterReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
-	_ = context.Background()
-	_ = r.Log.WithValues("harborcluster", req.NamespacedName)
+	ctx := context.Background()
+	log := r.Log.WithValues("harborcluster", req.NamespacedName)
 
-	// your logic here
+	var harborCluster goharborv1.HarborCluster
```

Comment:
```suggestion
	var harborCluster *goharborv1.HarborCluster
```
???

Reply:
I think, here needs a HarborCluster Object, not an address ref to HarborCluster Object.  Even if using a pointer,   you may use DeepCopy !

Reply:
You declare an object, but when using in the code, they are all ref point (& used), why not directly declare a pointer?